aptos-move/replay-benchmark/src/overrides.rs

```diff
@@ -73,6 +125,114 @@ impl OverrideConfig {
             state_override.insert(gas_schedule_state_key, gas_schedule_state_value);
         }
 
+        // Override packages.
+        let mut overridden_package_registries = HashMap::new();
+        for package in &self.package_override.packages {
+            // Modify existing package metadata or add new one.
+            let package_address = package
+                .modules()
+                .map(|m| m.self_addr())
```

Comment:
should they all be the same? does it make sense to assert?

Reply:
Yes, added an assert